pkg/cmd/build/build.go

```diff
@@ -283,6 +283,23 @@ func OptsFromBuildInfo(manifestName, svcName string, b *model.BuildInfo, o *type
 		file = extractFromContextAndDockerfile(b.Context, b.Dockerfile, svcName)
 	}
 
+	args := []model.BuildArg{}
+	optionsBuildArgs := map[string]string{}
+	for _, arg := range o.BuildArgs {
+		splittedArg := strings.SplitN(strings.Trim(arg, "\""), "=", 2)
+		optionsBuildArgs[splittedArg[0]] = splittedArg[1]
```

Comment:
we have to be careful here, if the user doesn't pass the value of the argument, it will throw a panic. To reproduce:

`okteto build --build-arg MY_BUILD_ARG`